src/Controls/tests/TestCases.HostApp/Issues/Issue25724.xaml.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	[Issue(IssueTracker.Github, 25724, "ObjectDisposedException When Toggling Header/Footer in CollectionView Dynamically", PlatformAffected.iOS)]
+
+	public partial class Issue25724 : ContentPage
```

Comment:
The build is failing. Review the nullable objects (header, footer). Can use `#nullable enable`.

```
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25724.xaml.cs(14,9): error CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
    6916 Warning(s)
    12 Error(s)

```

Reply:
Fixed the build issue by adding #nullable enable in the file. 
Please let me know if any further adjustments are needed.